packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1320,6 +1322,33 @@ describe("Runtime", () => {
 				);
 				assert.equal((runtime as unknown as { method2: () => unknown }).method2(), 42);
 			});
+
+			// A legacy partner team overrides the summarizeInternal method to add custom data to the Summary.
+			// Let's make sure we don't break them inadvertently, while we work to move them to a better pattern.
+			it("Preserve known overreach into private member", async () => {
+				const containerRuntime_withSummarizeInternal = (await ContainerRuntime.loadRuntime({
+					context: getMockContext() as IContainerContext,
+					registryEntries: [],
+					existing: false,
+					provideEntryPoint: mockProvideEntryPoint,
+				})) as unknown as {
+					summarizeInternal(
```

Comment:
Consider adding a comment to `summarizeInternal` function explaining that it should not be changed and why.